### DISCUSSION TITLE
Uninstall dev dependencies?

### DISCUSSION THREAD

**Discussion by cbini** [pdm-project/pdm#969] [Q&A] [answered]
Hi, I might be missing something obvious, but is there a way to reverse `pdm install`? I don't want to remove these packages from the pyproject.toml, but I accidentally installed dev dependencies on prod and was looking for a quick way to clean them up.

**Comment by cbini** [comment, marked as answer]
duh doi! just figured it out. delete `__pypackages__`

if you're installing to a virtualenv, you'll need to kill that I think

**Reply by robml** [reply]
Glad you figured it out. PDM is pretty straight forward that way, I was also confused when I first started with it, that just deleting the folder is enough usually.
